src/main/java/com/eprosima/fastdds/fastddsgen.java

```diff
@@ -512,6 +540,12 @@ public static void printHelp()
         System.out.println("\t\t-cs: IDL grammar apply case sensitive matching.");
         System.out.println("\t\t-test: executes FastDDSGen tests.");
         System.out.println("\t\t-python: generates python bindings for the generated types.");
+        System.out.print("\t\t-default_extensibility | -de <ext>: sets the default extensibility for types without");
+        System.out.println(" the @extensibility annotation.");
+        System.out.println("\t\t Values:");
+        System.out.println("\t\t\t* " + Annotation.final_str + " (default)");
```

Comment:
Default extensibility is appendable, not final.

Reply:
Done in 81dd83f